src/httpx_sse/_api.py

```diff
@@ -33,14 +34,18 @@ def iter_sse(self) -> Iterator[ServerSentEvent]:
             if sse is not None:
                 yield sse
 
-    async def aiter_sse(self) -> AsyncIterator[ServerSentEvent]:
+    async def aiter_sse(self) -> AsyncGenerator[ServerSentEvent]:
```

Comment:
I believe this should fix CI?

```suggestion
    async def aiter_sse(self) -> AsyncGenerator[ServerSentEvent, None]:
```